modules/checkbox/react/lib/Checkbox.tsx

```diff
@@ -31,6 +33,8 @@ const checkboxLabelDistance = spacing.xs;
 const checkboxWidth = 18;
 const rippleRadius = (spacing.l - checkboxWidth) / 2;
 
+const direction = Direction.RTL;
```

Comment:
This value should come from `CanvasProvider`

Reply:
Assuming that RTL exists in the `CanvasProvider`, will we be able to change it from a story?

Reply:
We won't unfortunately, but that's what the optional prop is for